src/routes/index.ts

```diff
@@ -1,14 +1,13 @@
 import { Request, Response, Router } from 'express';
+import userRoutes from './UserRoutes';
 import { responseSuccess } from '../utils/response.utils';
+
 const router = Router();
-router.get("/", (req: Request, res: Response) => {
-  res.send("Knights Ecommerce API");
-});
+
 router.get('/api/v1/status', (req: Request, res: Response) => {
   return responseSuccess(res, 202, 'This is a testing route that returns: 201');
 });
 
-// All routes should be imported here and get export after specifying first route
-// example router.use("/stock". stockRoutes) =>:: First import stockRoutes and use it here, This shows how the route export will be handled
+router.use('/user', userRoutes);
 
```

Comment:
@maxCastro1  since you updated this API status route, don't forget to update its test to ensure our workflow job passes smoothly, also seems '/user' is not functional, it might be best to remove it.

Reply:
yes , i have updated the routes for tests